kview/src/main/java/dev/ikm/komet/kview/mvvm/view/journal/JournalController.java

```diff
@@ -1216,7 +1216,7 @@ private void loadNavigationPanel(PublicIdStringKey<ActivityStream> navigationAct
                                 patternNavViewModel.setPropertyValue(VIEW_PROPERTIES, viewProperties)
                         );
         JFXNode<Pane, ConceptPatternNavController> conPatJFXNode = FXMLMvvmLoader.make(patternConceptConfig);
-        patternConceptNavigationPanel = conPatJFXNode.node();
+        patternConceptNavigationPanel = (StackPane) conPatJFXNode.node();
```

Comment:
Can we remove the down cast of a `StackPane`?
Can we declare `conPatJFXNode` like the following?
```java
JFXNode<StackPane, ConceptPatternNavController> conPatJFXNode = ...
```
This helps avoid a downcast.

Reply:
Good point that was a stupid miss from me. I've changed it. Thanks